src/server/rdb_load.cc

```diff
@@ -1606,6 +1649,15 @@ error_code RdbLoader::Load(io::Source* src) {
       return RdbError(errc::feature_not_supported);
     }
 
+    if (type == RDB_OPCODE_COMPRESSED_BLOB_START) {
+      RETURN_ON_ERR(HandleCompressedBlob());
+      continue;
+    }
+    if (type == RDB_OPCODE_COMPRESSED_BLOB_END) {
```

Comment:
I understand why it works, but I would like to understand why you need it. It's lightweight when decompressing but complicates things a bit when compressing. Lets chat.